client/src/components/UserEventsContainer.tsx

```diff
@@ -0,0 +1,57 @@
+import { EventFeedType } from "../lib/types";
+import { SESSION_KEY } from "../lib/constants";
+import axios from "axios";
+import { useQuery } from "react-query";
+import Loading from "./Loading";
+import React from "react";
+import { EventFeedContainer } from "./EventFeedContainer";
+
+interface Props {
+  username: string;
+  page: number;
+}
+
+const UserEventsContainer = ({ username, page }: Props) => {
+  const fetchPages = async () => {
+    const response = await axios.get(
+      `http://localhost:3001/events/${username}/${page - 1}`,
```

Comment:
the `page - 1` shouldn't be here, the display should be the one that modifies the value for example, 
you should also move this logic to the backend, so that the page result from the back is the same as the one you would use to request